data/whispering void/elenchus.txt

```diff
@@ -87,7 +87,7 @@ fleet "Devil-Run Miners"
 	cargo 0
 	personality
 		timid frugal mining harvests
-	variant 1
+	variant
```

Comment:
There are about 1050 instances of "variant" with no number in the game files, and just over 650 instances of "variant 1".
I'm not really interested in seeing four instances of "variant 1" changed to "variant" when nothing else is being done to the fleet definition.
If you want to change them all, we can consider that PR, but doing those 650 changes a few at a time in a bunch of unrelated PRs is far from my preference.